WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1830,6 +1827,24 @@ protected void onPostExecute(Uri uri) {
         }
     }
 
+    /**
+     * Media
+     */
+
+    private void addAllMedia(List<Uri> mediaUris) {
+        for (Uri mediaUri : mediaUris) {
+            if (mediaUri == null) {
```

Comment:
We should also ensure the toast only displays once, just in case more than one of the passed URIs is null or can't be added.